lib/loader.js

```diff
@@ -32,13 +32,42 @@ function getMatches (type, items, matches, component) {
   return imports
 }
 
-function install (install, content, imports) {
+function injectStylesSSR (imports) {
+  const styles = imports.map(componentImport => (componentImport[2] || [])).reduce((acc, styles) => {
+    styles && styles.forEach(style => acc.add(style))
+    return acc
+  }, new Set())
+
+  if (styles.size) {
+    return `
```

Comment:
This snippet should probably be moved to a new file in `src/runtime`

Reply:
The snippet is generated dynamically based on found styles that component uses. It will also contain `require` statements that need to be processed by webpack. So I don't think I can extract it exactly like the two existing files in `runtime` as a function that is just ran at runtime?

However I'm happy to extract this `injectStylesSSR ` function in a file if you think that'd make things cleaner 👍

Reply:
Oh yeah webpack doesn't really like dynamic require does it.